src/mandocvisitor.cpp

```diff
@@ -996,7 +996,8 @@ void ManDocVisitor::filter(const QCString &str)
       {
         case '.':  m_t << "\\&."; break; // see  bug652277
         case '\\': m_t << "\\\\"; break;
-        case '"':  c = '\''; // fall through
+        //case '"':  if(fixquotes)
```

Comment:
Looks like this lines is the only real change. The other changes do not have any effect as `fixquotes` doesn't appear to be used. 

Suggest to cleanup this PR.